### PR TITLE
Add cmdlets description in Hyper-V.md

### PR DESCRIPTION
**Pull request by velkovb**
Added description for module cmdlets and fixed links to include ./

### CODE REVIEW

**Comment by e0i** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

**Comment by e0i** [comment]
@dariomws306 @get-itips @tseward

Peer review needed for this PR. Thanks.

**Comment by get-itips** [comment]
Hello @velkovb  What is the reason behind adding the ./ ? current links on TOC file works. Thanks

**Comment by velkovb** [comment]
> 
> 
> Hello @velkovb What is the reason behind adding the ./ ? current links on TOC file works. Thanks

@get-itips Most of the other module articles follow this format and most probably this difference here is the cause that the actual descriptions are missing.

**Comment by get-itips** [comment]
> > Hello @velkovb What is the reason behind adding the ./ ? current links on TOC file works. Thanks
> 
> @get-itips Most of the other module articles follow this format and most probably this difference here is the cause that the actual descriptions are missing.

ok I don't think is necessary here, what you guys think @dariomws306 @tseward

File: docset/winserver2012-ps/hyper-v/Hyper-V.md [hunk 1]
```diff
@@ -14,495 +14,494 @@ ms.reviewer:
...
 
-### [Add-VMSwitch](Add-VMSwitch.md)
-{{Manually Enter Add-VMSwitch Description Here}}
+### [Add-VMSwitch](./Add-VMSwitch.md)
+Adds a virtual switch to an Ethernet resource pool.
 
-### [Checkpoint-VM](Checkpoint-VM.md)
-{{Manually Enter Checkpoint-VM Description Here}}
+### [Checkpoint-VM](./Checkpoint-VM.md)
+Creates a checkpoint of a virtual machine.
```
Comment (JohanFreelancer9) [inline docset/winserver2012-ps/hyper-v/Hyper-V.md]:
```suggestion
Creates a checkpoint for a virtual machine.
```

File: docset/winserver2012-ps/hyper-v/Hyper-V.md [hunk 2]
```diff
@@ -14,495 +14,494 @@ ms.reviewer:
...
 
-### [Restore-VMSnapshot](Restore-VMSnapshot.md)
-{{Manually Enter Restore-VMSnapshot Description Here}}
+### [Restore-VMSnapshot](./Restore-VMSnapshot.md)
+Restores a virtual machine checkpoint.
 
-### [Resume-VMReplication](Resume-VMReplication.md)
-{{Manually Enter Resume-VMReplication Description Here}}
+### [Resume-VMReplication](./Resume-VMReplication.md)
+Resumes a virtual machine replication that is in a state of Paused, Error, Resynchronization Required, or Suspended.
```
Comment (JohanFreelancer9) [inline docset/winserver2012-ps/hyper-v/Hyper-V.md]:
```suggestion
Resumes a virtual machine replication that is in a Paused, Error, Resynchronization Required, or Suspended state.
```

File: docset/winserver2012-ps/hyper-v/Hyper-V.md [hunk 3]
```diff
@@ -14,495 +14,494 @@ ms.reviewer:
...
 
-### [Resume-VM](Resume-VM.md)
-{{Manually Enter Resume-VM Description Here}}
+### [Resume-VM](./Resume-VM.md)
+Resumes a suspended (paused) virtual machine.
 
-### [Revoke-VMConnectAccess](Revoke-VMConnectAccess.md)
-{{Manually Enter Revoke-VMConnectAccess Description Here}}
+### [Revoke-VMConnectAccess](./Revoke-VMConnectAccess.md)
+Revokes access for one or more users to connect to a one or more virtual machines.
```
Comment (JohanFreelancer9) [inline docset/winserver2012-ps/hyper-v/Hyper-V.md]:
```suggestion
Revokes access for one or more users to connect to one or more virtual machines.
```

File: docset/winserver2012-ps/hyper-v/Hyper-V.md [hunk 4]
```diff
@@ -14,495 +14,494 @@ ms.reviewer:
...
 
-### [Suspend-VMReplication](Suspend-VMReplication.md)
-{{Manually Enter Suspend-VMReplication Description Here}}
+### [Suspend-VMReplication](./Suspend-VMReplication.md)
+Suspends replication of a virtual machine.
 
-### [Suspend-VM](Suspend-VM.md)
-{{Manually Enter Suspend-VM Description Here}}
+### [Suspend-VM](./Suspend-VM.md)
+Suspends, or pauses, a virtual machine.
```
Comment (JohanFreelancer9) [inline docset/winserver2012-ps/hyper-v/Hyper-V.md]:
```suggestion
Suspends or pauses a virtual machine.
```

**Comment by velkovb** [comment]
@get-itips when I was updating the descriptions, I actually did it via script and missing the ./ here and it being present in the other versions of the article complicated things a lot. I think we should have it for consistency with other version of the article.

**Comment by tseward** [comment]
Adding `./` doesn't change anything as that just means 'current directory'. It's extra keystrokes :-)

**Pull request merged**